sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -656,25 +672,37 @@
     .setDeleteHistoricalData(true)
     .create();
 
+  /**
+   * @deprecated since 5.2 soon to be removed
+   */
+  @Deprecated
   public static final String CONDITIONS_BY_LINE_KEY = "conditions_by_line";
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)